signature/signature.go

```diff
@@ -9,13 +9,17 @@ import (
 	"fmt"
 	"time"
 
+	internalSignature "github.com/containers/image/v5/signature/internal/signature"
 	"github.com/containers/image/v5/version"
 	digest "github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
 )
 
 const (
 	signatureType = "atomic container signature"
+
+	// Path pointing to a passphrase file when signing via GPGME.  Specified as an evironment variable.
+	PASSPHRASE_FILE_ENVIRONMENT = "CONTAINERS_SIGNATURE_PASSPHRASE_FILE"
```

Comment:
Nothing uses this AFAICS. Add a helper to `pkg/cli/environment`, and move the constant there (or just don’t make it public)?

Reply:
It's not used/needed anymore, so we can safely remove it.